pkg/resources/digitalocean/resources.go

```diff
@@ -195,7 +196,8 @@ func listDNS(cloud fi.Cloud, clusterName string) ([]*resources.Resource, error)
 	}
 
 	if domainName == "" {
-		return nil, fmt.Errorf("failed to find domain for cluster: %s", clusterName)
+		klog.Info("Domain Name is empty. Possible that the cluster is configured as gossip..")
```

Comment:
we could check here does the `clusterName` endswith k8s.local. Then we can say is it gossip cluster or not. If not then we should return real error